### PR TITLE
added support for isso: http://posativ.org/isso/

### PR DESCRIPTION
**Pull request by dertuxmalwieder**

### CODE REVIEW

File: plugins/isso.lisp [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  "<div class=\"comments\">
+      <section id=\"isso-thread\"></section>
+      <script data-isso=\"~a/\"
+              src=\"~a/js/embed.min.js\"></script>
+    </div>")
+
+(defun enable (&key isso-url)
+  (flet ((inject-p (x)
+           (when (typep x 'post)
+             (format nil *isso-header* isso-url))))
```
Comment (kingcons) [inline plugins/isso.lisp]: There is actually a bug here as your format string `*isso-header*` takes two arguments but you are only passing it one argument `isso-url`. I believe you should be calling `(format nil *isso-header* isso-url isso-url)`.

Comment (dertuxmalwieder) [inline plugins/isso.lisp]: Oh, yes, of course. Sorry. Do you need a second pull request?

Comment (kingcons) [inline plugins/isso.lisp]: Just another commit to fix it will be fine.

**Comment by dertuxmalwieder** [comment]
Now...

**Comment by kingcons** [comment]
@dertuxmalwieder This has been rebased and merged into master. Thanks! :)

**Comment by dertuxmalwieder** [comment]
Thanks too!

**Pull request closed without merging**